src/javascript/JContent/ContentRoute/ContentLayout/ContentTable/reactTable/columns/columns.js

```diff
@@ -79,9 +80,17 @@ export const allColumnData = [
         Header: '',
         Cell: CellVisibleActions,
         width: '60px'
+    },
+    {
+        id: 'usages',
+        label: 'jcontent:label.contentManager.deleteAction.infoTable.headerLabel',
+        Header: Header,
+        Cell: CellUsages,
+        sortable: false,
+        width: '100px'
     }
 ];
 
-export const reducedColumnData = allColumnData.filter(c => c.id !== 'type');
-
-export const deletionInfoColumnData = allColumnData.filter(c => ['name', 'status', 'type', 'publicationStatus'].includes(c.id)).map(c => ({...c, sortable: false}));
+export const mainColumnData = allColumnData.filter(c => !['usages'].includes(c.id));
+export const reducedColumnData = allColumnData.filter(c => !['type', 'usages'].includes(c.id));
+export const deletionInfoColumnData = allColumnData.filter(c => ['name', 'status', 'type', 'publicationStatus', 'usages'].includes(c.id)).map(c => ({...c, sortable: false}));
```

Comment:
Can't we have a simple list like `mainColumnData = [ publicationStatus, selection, name, ...]` ? that would be much readable than all these filtered lists. Also `allColumnData` is not really useful as a list anymore

Reply:
I did not meant duplicating entries definitions, sorry .. just pushed what i had i mind